src/App.jsx

```diff
@@ -1,7 +1,25 @@
 import "./App.css";
+import Header from "./components/Header";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import NavBar from "./components/navbar";
```

Comment:
I was getting an error when trying to run this at first. Check out the NavBar import spelling of the file 👀